modules/tests/test/dataobject/operations.go

```diff
@@ -32,9 +42,29 @@ func WaitForSuccessfulDataSource(cdiClientSet cdicliv1beta1.CdiV1beta1Interface,
 	})
 }
 
-func IsDataVolumeImportSuccessful(cdiClientSet cdicliv1beta1.CdiV1beta1Interface, namespace string, name string) bool {
+func WaitForBoundPVC(k8sClient *kubernetes.Clientset, namespace, name string, timeout time.Duration) error {
+	return wait.PollImmediate(constants.PollInterval, timeout, func() (bool, error) {
```

Comment:
Can we use Eventually, as this is test code?

Reply:
Any particular reason why changing this?

Reply:
IMO leveraging the test framework is not a bad idea, but functionally it should be equal.

Reply:
the function is no longer needed and was deleted